src/Annotation/README.md

```diff
@@ -0,0 +1,236 @@
+# Annotation feature
+
+This feature provides a system to declare annotations in docComment and 
+retrieve it into php classes.
+
+## Declaration 
+_Rules to be observed when declaring an annotation in the docComment_
+##### Identification
+The identification character of an annotation **MUST** be an `@`  
+e.g. `@Route`
+
+##### Annotation name
+The annotation name:
+- **MUST** start width uppercase. \[A-Z\].
+- Can only use alpha-numeric characters. [a-zA-Z]. It can't use special
+  characters or white spaces.   
+  
+e.g. `@MyAnnotationName`  
+_This option can be changed by changing the default annotation name declaration regex_
+
+  
+##### Options
+###### Declaration
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (6:1)](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6283010)